sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -134,4 +136,24 @@ export class ApplicationController extends BaseController {
 
     return assessment;
   }
+
+  /**
+   * Confirm Assessment of a Student.
+   * @param applicationId application id to be updated.
+   */
+  @AllowAuthorizedParty(AuthorizedParties.student)
+  @Patch(":applicationId/confirm-assessment")
```

Comment:
The controller or the service should check at some point if the application being updated belongs to the students.